File: Day_7/Q25.cpp
#include <iostream>
using namespace std;
//Calculating Factorial
int fact(int n){
    if(n == 0 || n == 1) return 1;
    return(n*fact(n-1));
}
    int main(){
        int n;
        cin >> n;
        //Validating Input
        if(n < 0 ) {
            cout << "Invalid Input";
            return 0;
        }
        //Printing Result
        cout << fact(n);
        return 0;
    }

File: Day_7/Q25.java
import java.util.Scanner;

class Q25{
    //Calculating Factorial
    static int fact(int n){
        if(n == 0 || n == 1) return 1;
        return(n*fact(n-1));
    }
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        //Validating Input
        if(n < 0) System.out.print("Invalid Input");
        //Printing Result
        else{
            System.out.print(fact(n));
        }
        sc.close();
    }
}